salt/modules/virt.py

```diff
@@ -6744,17 +7148,152 @@ def network_define(name, bridge, forward, ipv4_config=None, ipv6_config=None, **
     :param username: username to connect with, overriding defaults
     :param password: password to connect with, overriding defaults
 
+    :param mtu: size of the Maximum Transmission Unit (MTU) of the network.
+        (default ``None``)
+
+        .. versionadded:: Aluminium
+
+    :param domain: DNS domain name of the DHCP server.
+        The value is a dictionary with a mandatory ``name`` property and an optional ``localOnly`` boolean one.
+        (default ``None``)
+
+        .. code-block:: yaml
+
+          - domain:
+              name: lab.acme.org
+              localOnly: True
+
+        .. versionadded:: Aluminium
+
+    :param nat: addresses and ports to route in NAT forward mode.
+        The value is a dictionary with optional keys ``address`` and ``port``.
+        Both values are a dictionary with ``start`` and ``end`` values.
+        (default ``None``)
+
+        .. code-block:: yaml
+
+          - forward: nat
+          - nat:
+              address:
+                start: 1.2.3.4
+                end: 1.2.3.10
+              port:
+                start: 500
+                end: 1000
+
+        .. versionadded:: Aluminium
+
+    :param interfaces: whitespace separated list of network interfaces devices that can be used for this network.
+        (default ``None``)
+
+        .. code-block:: yaml
+
+          - forward: passthrough
+          - interfaces: "eth10 eth11 eth12"
+
+        .. versionadded:: Aluminium
+
+    :param addresses: whitespace separated list of addreses of PCI devices that can be used for this network in `hostdev` forward mode.
+        (default ``None``)
+
+        .. code-block:: yaml
+
+          - forward: hostdev
+          - interfaces: "0000:04:00.1 0000:e3:01.2"
+
+        .. versionadded:: Aluminium
+
+    :param physical_function: device name of the physical interface to use in ``hostdev`` forward mode.
+        (default ``None``)
+
+        .. code-block:: yaml
+
+          - forward: hostdev
+          - physical_function: "eth0"
+
+        .. versionadded:: Aluminium
+
+    :param dns: virtual network DNS configuration
+        The value is a dictionary described in net-define-dns_.
+        (default ``None``)
+
+        .. code-block:: yaml
+
+          - dns:
+              forwarders:
+                - domain: example.com
+                  addr: 192.168.1.1
+                - addr: 8.8.8.8
+                - domain: www.example.com
+              txt:
+                host.widgets.com.: "printer=lpr5"
```

Comment:
**question**: Should this also be something like `widgets.example.com`?

Reply:
this is purely left to dnsmasq... so I guess that can also be it

Reply:
I only raise the issue because I read somewhere recently that example documentation should be examples :upside_down_face: 

Basically -- putting "real" things in increases the chances that someone accidentally does a copy/paste and horks something up. Keeping things as example domains, [IP addresses](https://tools.ietf.org/html/rfc5737), etc, helps to avoid that problem. If it were the only thing on a PR I probably wouldn't bring it up, but I was already in here thinking :sweat_smile:

Reply:
I'm no DNS expert... so I'll put there the values you think are best. IIRC I picked that one from some libvirt example file

Reply:
I'll add the examples from https://github.com/imp/dnsmasq/blob/master/dnsmasq.conf.example to be on the safe side